cmd/server/tpm_attestor/tpm_attestor.go

```diff
@@ -12,26 +12,22 @@
  ** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ** See the License for the specific language governing permissions and
  ** limitations under the License.
- */
+*/
 
 package main
 
 import (
 	"bytes"
 	"context"
-	"crypto/sha256"
-	"encoding/asn1"
-	"encoding/base64"
 	"encoding/json"
 	"errors"
 	"fmt"
-	"log"
-
-	"github.com/google/certificate-transparency-go/x509"
+	"io/ioutil"
+	"path/filepath"
 
 	"github.com/bloomberg/spire-tpm-plugin/pkg/common"
+	"github.com/google/certificate-transparency-go/x509"
```

Comment:
same here, did you mean `encoding/x509`?

Reply:
see above